design-documents/service-isolation/invoker.md

```diff
@@ -0,0 +1,122 @@
+### Invoker API and SPI
+Responsible for contacting remote services
+#### API
+##### Remote call response
+Result of executing the call.
+Can be used by 3rd party devs to add data to remote calls results or read it when they choose to modify
+remote calling mechanism.
+```php
+interface InvokeResponseInterface {
```

Comment:
Are we going to have many response interfaces in the library? Can be just ResponseInterface probably.

Reply:
don't want to mix it with actual network responses